### PR TITLE
edited queries for category grouping

### PR DESCRIPTION
**Pull request by JonLev03-hub**

### CODE REVIEW

File: app/Http/Controllers/Controller.php [hunk 1]
```diff
@@ -46,26 +46,28 @@ public function index()
...
-        $kitchenItems = Listing::latest()->where('status', '!=', 'Sold')->orWhere('category', 'LIKE', '%kitchen%')->limit(10)->get();
-        $schoolItems = Listing::latest()->where('status', '!=', 'Sold')->orWhere('category', 'LIKE', '%school accesories%')->limit(10)->get();
-        $bookItems = Listing::latest()->where('status', '!=', 'Sold')->orWhere('category', 'LIKE', '%sbooks%')->limit(10)->get();
+        $furnitureItems = Listing::latest()->where('status', '!=', 'Sold')->Where('category', 'LIKE', '%furniture%')->limit(10)->get();
+        $clothesItems = Listing::latest()->where('status', '!=', 'Sold')->where('category', 'LIKE', '%clothes%')->limit(10)->get();
+        $electronicsItems = Listing::latest()->where('status', '!=', 'Sold')->where('category', 'LIKE', '%electronics%')->limit(10)->get();
+        $kitchenItems = Listing::latest()->where('status', '!=', 'Sold')->where('category', 'LIKE', '%kitchen%')->limit(10)->get();
+        $schoolItems = Listing::latest()->where('status', '!=', 'Sold')->where('category', 'LIKE', '%school accesories%')->limit(10)->get();
```
Comment (ayubshahab) [inline app/Http/Controllers/Controller.php:53]: There seems to be a typo here, misspelling of "school accessories". Please fix the spelling mistake I made. Thanks

File: app/Http/Controllers/Controller.php [hunk 2]
```diff
@@ -46,26 +46,28 @@ public function index()
...
+        $bookItems = Listing::latest()->where('status', '!=', 'Sold')->where('category', 'LIKE', '%sbooks%')->limit(10)->get();
```
Comment (ayubshahab) [inline app/Http/Controllers/Controller.php:55]: Another typo, "books" is spelled incorrectly. Please fix the spelling mistake I made. Thanks

File: app/Http/Controllers/Controller.php [hunk 3]
```diff
@@ -46,26 +46,28 @@ public function index()
...
-        $kitchenRent = Rentable::latest()->where('status', '!=', 'Rented')->orWhere('category', 'LIKE', '%kitchen%')->limit(10)->get();
-        $schoolRent = Rentable::latest()->where('status', '!=', 'Rented')->orWhere('category', 'LIKE', '%school accesories%')->limit(10)->get();
-        $bookRent = Rentable::latest()->where('status', '!=', 'Rented')->orWhere('category', 'LIKE', '%sbooks%')->limit(10)->get();
+        $furnitureRent = Rentable::latest()->where('status', '!=', 'Rented')->where('category', 'LIKE', '%furniture%')->limit(10)->get();
+        $clothesRent = Rentable::latest()->where('status', '!=', 'Rented')->where('category', 'LIKE', '%clothes%')->limit(10)->get();
+        $electronicsRent = Rentable::latest()->where('status', '!=', 'Rented')->where('category', 'LIKE', '%electronics%')->limit(10)->get();
+        $kitchenRent = Rentable::latest()->where('status', '!=', 'Rented')->where('category', 'LIKE', '%kitchen%')->limit(10)->get();
+        $schoolRent = Rentable::latest()->where('status', '!=', 'Rented')->where('category', 'LIKE', '%school accesories%')->limit(10)->get();
```
Comment (ayubshahab) [inline app/Http/Controllers/Controller.php:61]: Same issue as above, fix "school accessories" spelling mistake. Thanks

File: app/Http/Controllers/Controller.php [hunk 4]
```diff
@@ -46,26 +46,28 @@ public function index()
...
+        $bookRent = Rentable::latest()->where('status', '!=', 'Rented')->where('category', 'LIKE', '%sbooks%')->limit(10)->get();
```
Comment (ayubshahab) [inline app/Http/Controllers/Controller.php:62]: same issue as above

**Comment by ayubshahab** [comment]
Overall the changes look good, just a spelling mistake. Haha laravel query builder is rather annoying. What exactly is the difference between orWhere and where? seems like I had the wrong one here previously.

**Comment by JonLev03-hub** [comment]
> Overall the changes look good, just a spelling mistake. Haha laravel query builder is rather annoying. What exactly is the difference between orWhere and where? seems like I had the wrong one here previously.

where means the condition needs to be met, orWhere means one of the surrounding conditions needs to be met. So previously the filter was saying "if the item isnt sold OR the category matches" now its "if the item isn't sold AND the category matches"

**Comment by JonLev03-hub** [comment]
I hadn't noticed the typos since I didn't change that end of the code I assumed there were just no items of those categories. I can fix that before the meeting tonight. and make a new pull request

**Pull request closed without merging**